Views/Library/ZimFilesOpened.swift

```diff
@@ -46,6 +47,9 @@ struct ZimFilesOpened: View {
                 Message(text: "zim_file_opened.overlay.no-opened.message".localized)
             }
         }
+        .onChange(of: zimFiles.count) { _ in
+            viewModel.selectedZimFile = zimFiles.first // makes sure we also nil out, if all ZIMs were unlinked 
+        }
```

Comment:
Explanation: `zimFiles.first` returns an optional, so it covers both cases, when there's still files to pick from, and when there's nothing left.